Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2689,22 +2708,15 @@ def slack_get_integration_context():
         questions = json.loads(integration_context.get('questions'))
         context_statistics['Questions Count'] = len(questions)
         context_statistics['Questions Size In Bytes'] = sys.getsizeof(integration_context.get('questions', []))
-    readable_stats = tableToMarkdown(name='Long Running Context Statistics', t=context_statistics)
-    demisto.results({
-        'Type': entryTypes['note'],
-        'HumanReadable': readable_stats,
-        'ContentsFormat': EntryFormat.MARKDOWN,
-        'Contents': readable_stats,
-    })
-    return_results(fileResult('slack_integration_context.json', json.dumps(integration_context), EntryType.ENTRY_INFO_FILE))
+    return context_statistics, integration_context
 
 
 def main() -> None:
     """
     Main
     """
     global CLIENT
-    if is_debug_mode():
```

Comment:
So uh fun fact. This doesn't actually fire in long-running since we don't push the debug-mode flag into the container.